test/sortitionTreeTest.js

```diff
@@ -1,218 +1,275 @@
-      await sortition.publicInsertOperator(alice, 0x1234)
-      await sortition.publicInsertOperator(bob, 0x9876)
+      await sortition.publicInsertOperator(alice.address, 0x1234)
+      await sortition.publicInsertOperator(bob.address, 0x9876)
 
-      await sortition.publicRemoveOperator(alice)
-      const deletedLeaf = await sortition.getLeaf.call(0x00000)
-      assert.equal(deletedLeaf, 0)
+      await sortition.publicRemoveOperator(alice.address)
+      const deletedLeaf = await sortition.getLeaf(0x00000)
+      expect(deletedLeaf).to.be.equal(0)
 
-      await sortition.publicInsertOperator(alice, 0xdead)
+      await sortition.publicInsertOperator(alice.address, 0xdead)
 
-      const stillDeletedLeaf = await sortition.getLeaf.call(0x00000)
-      assert.equal(stillDeletedLeaf, 0)
+      const stillDeletedLeaf = await sortition.getLeaf(0x00000)
+      expect(stillDeletedLeaf).to.be.equal(0)
 
-      const root = await sortition.getRoot.call()
-
-      assert.equal(toHex(root), "0x17723")
+      const root = await sortition.getRoot()
+      expect(ethers.utils.hexlify(root)).to.be.equal("0x017723")
     })
   })
 
   describe("leaf selection", async () => {
     it("works as expected", async () => {
-      await sortition.publicInsertOperator(alice, 451)
-      await sortition.publicInsertOperator(bob, 1984)
+      await sortition.publicInsertOperator(alice.address, 451)
+      await sortition.publicInsertOperator(bob.address, 1984)
       const index1 = 450
       const index2 = 451
 
-      const position1 = await sortition.publicPickWeightedLeaf.call(index1)
-      assert.equal(position1, 0)
+      const position1 = await sortition.publicPickWeightedLeaf(index1)
+      expect(position1).to.be.equal(0)
 
-      const leaf1 = await sortition.getLeaf.call(position1)
-      const address1 = await sortition.leafAddress.call(leaf1)
-      assert.equal(address1, alice)
+      const leaf1 = await sortition.getLeaf(position1)
+      const address1 = await sortition.leafAddress(leaf1)
+      expect(address1).to.be.equal(alice.address)
 
-      const position2 = await sortition.publicPickWeightedLeaf.call(index2)
-      assert.equal(position2, 1)
+      const position2 = await sortition.publicPickWeightedLeaf(index2)
+      expect(position2).to.be.equal(1)
 
-      const leaf2 = await sortition.getLeaf.call(position2)
-      const address2 = await sortition.leafAddress.call(leaf2)
-      assert.equal(address2, bob)
+      const leaf2 = await sortition.getLeaf(position2)
+      const address2 = await sortition.leafAddress(leaf2)
+      expect(address2).to.be.equal(bob.address)
     })
   })
 
-  describe("operatorsInPool()", async () => {
-    it("works as expected", async () => {
-      await sortition.publicInsertOperator(alice, 1)
-      const nOperators = await sortition.operatorsInPool.call()
-      assert.equal(nOperators, 1)
+  describe("operatorsInPool", async () => {
+    context("when there is an operator is in the pool", () => {
```

Comment:
"when the operator is in the pool" ? 😄

Reply:
Done in https://github.com/keep-network/sortition-pools/pull/127/commits/1fdcda0aae9ee9065e524e6280f2dc2157e2aaf3.